features/integrate_build_system/waf-installer/LICENSE

```diff
@@ -0,0 +1,25 @@
+Redistribution and use in source and binary forms, with or without
```

Comment:
I think this license is not needed

Reply:
I think that it is needed because of the point 2: "must reproduce the above copyright notice"

Reply:
you are right, sorry!